### ISSUE TITLE
How to share HttpApi.Client Project/library to 3rd-party clients

### ISSUE THREAD

**Issue by ertugrulbakidal** [abpframework/abp#17059]
* ABP Framework version: 7.1.1
* Database provider: EF Core

Hi,
In the Abp documentation described that ```You can share this library to 3rd-party clients``` I understand that I can reference HttpApi.Client Project in my client app like console, wpf, shortly any C# client app. But how can I use HttpApi.Client Project in an asp.net core app which is in another server? I can not reference HttpApi.Client Project because my client app is another server than Abp app. Do I need to create a nuget package ? What are the possible solutions ?

**Comment by maliming** [comment]
hi

The 3rd-party has to be an abp project.

**Comment by ertugrulbakidal** [comment]
@maliming  yes my 3rd-party app is an abp project,too. But as my main abp app and client abp app are in different servers, I don't know how to use HttpApi.Client project. Do I need to create a nuget package?

**Comment by maliming** [comment]
> Do I need to create a nuget package?

No, There is no difference with `HttpApi.Client.ConsoleTestApp`.
